emulator/blockchain.go

```diff
@@ -555,8 +567,12 @@ func configureFVM(blockchain *Blockchain, conf config, blocks *blocks) (*fvm.Vir
 		CapabilityControllersEnabled: true,
 		CoverageReport:               conf.CoverageReport,
 	}
+	rt := conf.runtime
+	if conf.runtime == nil {
```

Comment:
```suggestion
	if rt == nil {
```

Reply:
Good point :+1: 
Fixed in https://github.com/onflow/flow-emulator/pull/476/commits/27408cfe3291fef240233369708d11933d1a33d8